test/tests/interfaces/pbs_preempt_params.py

```diff
@@ -109,38 +114,12 @@ def test_set_unset_preempt_prio(self):
         values and checks if the server allows/disallows the operation.
         """
         msg = "preempt_prio set to invalid value"
-        a = {'preempt_prio': 'abc'}
-        try:
-            self.server.manager(MGR_CMD_SET, SCHED, a,
-                                runas=ROOT_USER)
-        except PbsManagerError as e:
-            self.assertTrue(self.error_illegal in e.msg[0])
-            self.assertTrue(self.error_code_illegal in e.msg[1])
-        else:
-            self.fail(msg)
+        param = 'preempt_prio'
 
-        a = {'preempt_prio': '123abc'}
-        try:
-            self.server.manager(MGR_CMD_SET, SCHED, a,
-                                runas=ROOT_USER)
-        except PbsManagerError as e:
-            self.assertTrue(self.error_illegal in e.msg[0])
-            self.assertTrue(self.error_code_illegal in e.msg[1])
-        else:
-            self.fail(msg)
-
-        a = {'preempt_prio': 'abc123'}
-        try:
-            self.server.manager(MGR_CMD_SET, SCHED, a,
-                                runas=ROOT_USER)
-        except PbsManagerError as e:
-            self.assertTrue(self.error_illegal in e.msg[0])
-            self.assertTrue(self.error_code_illegal in e.msg[1])
-        else:
-            self.fail(msg)
+        self.common_tests(param, msg)
 
         p = '"express_queue, nrmal_jobs, server_softlimits, queue_softlimits"'
```

Comment:
Typo for normal_jobs

Reply:
hey @Klovely - this is not a typo - I am testing to make sure that preempt_prio is not set to this value.